src/lib/components/ILSMenu/ILSMenu.js

```diff
@@ -115,6 +116,13 @@ class ILSMenu extends Component {
                   </div>
                 </Menu.Item>
                 <Menu.Menu position="right">
+                  <Menu.Item
+                    onClick={() =>
```

Comment:
In general we should use `Link` when creating links, for accessibility reasons, instead of relying on events. For instance here it's not possible to open it in a new tab, while I'm sure some users may want to do so.